src/SFA.DAS.Forecasting.Domain.UnitTests/Estimations/AccountEstimationTests.cs

```diff
@@ -144,5 +147,17 @@ public void Add_Virtual_Apprenticeship_Assigns_FundingSource_To_Apprenticeship()
             var apprenticeship2 = estimation.AddVirtualApprenticeship("course-1", "test course", 1, 1, 2019, 5, 18, 1000, FundingSource.Levy);
             Assert.AreEqual(apprenticeship2.FundingSource, FundingSource.Levy);
         }
+
+        
+        [TestCase(-20, 12, false)]
+        [TestCase(-15, 12, false)]
+        [TestCase(-5, 10, true)]
+        public void Has_Valid_Apprenticeships_Returns_False_When_No_Active_Apprenticeshp(int deductNumberOfMonths, int numberOfMonths, bool isValid)
+        {
+            var estimation = ResolveEstimation();
+            var startDate = DateTime.Now.AddMonths(deductNumberOfMonths);
+            var apprenticeship = estimation.AddVirtualApprenticeship("course-1", "test course", 1, startDate.Month, startDate.Year, 5, numberOfMonths, 1000, FundingSource.Transfer);
```

Comment:
we are not asserting the response from this call. do we need.

Reply:
No we just want to add an apprenticeship, it is part of setup.